website/src/pages/index.tsx

```diff
@@ -117,7 +117,7 @@ function Hero() {
           <h1 className="title-font sm:text-4xl text-3xl lg:text-6xl mb-4 font-medium text-gray-900 dark:text-white">
             Containers and Kubernetes for application developers
           </h1>
-          <p className="text-base text-gray-700 dark:text-gray-500 md:text-lg">
+          <p className="text-base text-gray-700 md:text-lg">
```

Comment:
`text-gray-700` is supposed to be giving the color to the paragraph. 
Here, by overriding the class, you are defining the color in another place and it's not following the "tailwind" practices.

I think it would be easier to switch the color of the paragraph to `text-gray-100` or something like that. 
You can check the colors provided out of the box by tailwind here: https://tailwindcss.com/docs/text-color